### PR TITLE
Feature/corrector

### PR DESCRIPTION
**Pull request by Tchilo**
- Add corrector file
- Create corrector class
- Add corrector function 
- Add an instance of corrector class in person

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,79 @@
...
+
+### Prerequisites
+```
+Ruby
+```
+
+### Setup
+```
+git clone git@github.com:Tchilo/school-library.git
+cd school-library.git
```
Comment (TedLivist) [inline README.md]: - [x] You may want to fix this documentation error here. Users cannot really set up your app using this SSH link. You appear to be missing the `-oop`. The correct SSH link is this: **`git@github.com:Tchilo/school-library-oop.git`**, not what you have here.

File: README.md [hunk 2]
```diff
@@ -1 +1,79 @@
...
+```
+
+### Run Program
+```
+ruby school-library.rb
```
Comment (TedLivist) [inline README.md]: - [x] Your app has no entry point yet and there is really no `school-library` file in your app, so I'd suggest you remove this or use some other file that exists (as some kind of placeholder).

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Tchilo,

Good job so far!
✔️ Your `correct_name` method works just fine
✔️ You have your `validate_name` method well set up
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Tchilo) [reply README.md]: Thanks, I forgot the -oop 😄

File: README.md (hunk 2 above)
Comment (Tchilo) [reply README.md]: Thanks I will change that. 😄

**Review by rukundoeric — APPROVED** [review]
Hi @Tchilo,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**